### DISCUSSION TITLE
valor e trabalho

### DISCUSSION THREAD

**Discussion by ghost** [DesignLiquido/delegua#676] [P&R] [answered]

**Comment by leonelsanchesdasilva** [comment, marked as answer]
Olá, @lucasnuic 👋 

Sim, precisamos de toda a divulgação que pudermos. Obrigado pela contribuição!

**Reply by vgeruso** [reply]
Tmj, se precisar de contibuição da minha parte para o delegua, estarei aqui, vou buscar conhecer o projeto e tentar fazer minha parte

**Reply by vgeruso** [reply]
Eu gostaria até de saber como eu posso fazer parte e contribuir para o projeto, já que eu sou dev JS/TS acho que posso ajudar.

**Reply by leonelsanchesdasilva** [reply]
@vgeruso Há várias formas de contribuir, seja no código em si, melhorando a documentação, divulgando em redes sociais (artigos, vídeos, posts) e testando código e abrindo bugs. Temos até recompensas em dólar para issues com as respectivas etiquetas. 

Temos também um servidor de Discord, aberto a todos: https://discord.gg/4tBxWSSbdV
